_episodes/13-hosting.md

```diff
@@ -42,7 +42,7 @@ communities already using the same service.
 
 As an example, Software Carpentry [is on
 GitHub](https://github.com/swcarpentry/) where you can find the [source for this
-page](https://github.com/swcarpentry/git-novice/blob/gh-pages/13-hosting.md).
+page](https://github.com/swcarpentry/git-novice/blob/gh-pages/_episodes/13-hosting.md).
```

Comment:
Good catch! But we prefer to use the variables defined in `_config.yml` for this type of URLs, so we don't get broken links so when we move things around :)

`{{ page.root }}` equals the root dir of the repo. So you should do `...[source for this page]({{page.root}}/_episodes/13-hosting.md`